static/app/views/issueDetails/groupEventCarousel.tsx

```diff
@@ -337,20 +381,37 @@ export function GroupEventCarousel({event, group, projectSlug}: GroupEventCarous
           ]}
         />
         {xlargeViewport && (
-          <Button size={BUTTON_SIZE} onClick={copyLink}>
-            Copy Link
+          <Button
+            title={
+              isHelpfulEventUiEnabled ? t('Copy link to this issue event') : undefined
+            }
+            size={BUTTON_SIZE}
+            onClick={copyLink}
+            icon={isHelpfulEventUiEnabled ? <IconLink /> : undefined}
+          >
+            {!isHelpfulEventUiEnabled && 'Copy Link'}
```

Comment:
We do want to have an `aria-label` here when there is just an icon. Otherwise screen readers won't know what this button does. (the tooltip doesn't help in that regard)

Reply:
Should the EventNavigationButtons also have aria-labels then?

Reply:
Yes, but I think they already have them